action.go

```diff
@@ -0,0 +1,18 @@
+package brokerTags
+
+// Action - Custom type to hold value for broker action
+type Action int
+
+const (
+	createdAtTagKey = "Created at"
```

Comment:
This looks like an "implement enums in Go" attempt to me. I think you could simplify by using string consts like you did in tags.go — you'll drop the custom type (though you could even keep that if you want, just to alias `string`), the map between ints and strings, the method, and the corresponding tests.

Reply:
Resolution: The way this is used, you want to call the name of an action, you internally want to map that action to a string like "Created at", "Updated at". Since that's more of an internal detail, we're keeping it as-is.